### PR TITLE
merged pib-beta-fleet

### PR DESCRIPTION
**Pull request by bartr**
## Purpose of PR

- merged work from pib-beta-fleet
- scrub repos for akdc
- scrub repos for retail edge
- standardize app.yaml

## Type of PR

- [x] Documentation changes
- [x] Code changes
- [x] Test changes
- [x] CI-CD changes
- [ ] GitHub Template changes

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/pib-cli/issues/452
- Closes https://github.com/retaildevcrews/pib-cli/issues/453
- Closes https://github.com/retaildevcrews/pib-cli/issues/630
- Closes https://github.com/retaildevcrews/pib-cli/issues/631
- Closes https://github.com/retaildevcrews/pib-cli/issues/632

### CODE REVIEW

File: .devcontainer/on-create.sh [hunk 1]
```diff
@@ -52,17 +60,43 @@ mkdir -p "$HOME/.oh-my-zsh/completions"
...
 
     echo ""
     echo "compinit"
 
 } >> "$HOME/.zshrc"
 
+# echo "dowloading kic CLI"
+cd cli || exit
+tag="0.8.3"
+wget -O kic.tar.gz "https://github.com/retaildevcrews/pib-cli/releases/download/$tag/kic-$tag-linux-amd64.tar.gz"
```
Comment (aflinchb) [inline .devcontainer/on-create.sh:73]: Do we want kic + inner loop cluster in this repo?

File: apps/flux-system/app.yaml [hunk 2]
```diff
@@ -3,9 +3,9 @@ deployments:
   dev:
     target: dev
     values:
-      name: flux-system
+      app: flux-system
       namespace: flux-system
       workspace: flux-system
-      repo: yourTenant/yourRepo
+      repo: retaildevcrews/pib-beta-fleet
       branch: main
```
Comment (aflinchb) [inline apps/flux-system/app.yaml:10]: Should we leave this as yourTenant/yourRepo so this repo isn't used if the readme isn't followed closely?

File: apps/heartbeat/.gitops/dev/ingressHttp.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{gitops.app}}
+  namespace: {{gitops.namespace}}
```
Comment (aflinchb) [inline apps/heartbeat/.gitops/dev/ingressHttp.yaml:5]: For consistency, can we either hardcode "heartbeat" for name and namespace in the ingress files or update the deployment.yaml and service.yaml to use the app metadata.

File: .devcontainer/on-create.sh (hunk 1 above)
Comment (bartr) [reply .devcontainer/on-create.sh:73]: I think so but with the new kic install fleet, maybe not?

Let's leave it in this PR and track on the board.

**Review by aflinchb — APPROVED** [review]
Approved with a few comments (most can be addressed in another pr for easier review)

File: apps/heartbeat/.gitops/dev/ingressHttp.yaml (hunk 3 above)
Comment (bartr) [reply apps/heartbeat/.gitops/dev/ingressHttp.yaml:5]: @aflinchb - good catch - fixed

added issue to board to use pib-service for heartbeat deployment

File: sample-apps/badapp/.gitops/dev/badapp.yaml [hunk 4]
```diff
@@ -53,13 +53,13 @@ spec:
...
-    app: {{gitops.name}}
+    app: {{gitops.app}}
 
 ---
 apiVersion: traefik.containo.us/v1alpha1
 kind: IngressRoute
 metadata:
-  name: {{gitops.name}}-https
+  name: {{gitops.app}}-https
   namespace: default
```
[the remark this answers is not in the document]
Comment (bartr) [reply sample-apps/badapp/.gitops/dev/badapp.yaml:63]: we have an issue on the board to update all the apps in sample-apps

**Pull request merged**